tests/integration/components/test_render_cookies_banner.py

```diff
@@ -0,0 +1,10 @@
+from tests.integration.integration_test_case import IntegrationTestCase
```

Comment:
File name needs updating. And just making sure that this test only covers the content and not the JS behaviour as if that broke, we still would be able to tell.